crates/common/src/class_definition.rs

```diff
@@ -8,7 +8,13 @@ use serde::{Deserialize, Serialize};
 use serde_json::value::RawValue;
 use serde_with::serde_as;
 
-use crate::{ByteCodeOffset, EntryPoint};
+use crate::{ByteCodeOffset, ClassHash, EntryPoint};
+
+#[derive(Debug)]
+pub struct ClassDefinitionWithHash<'a> {
+    pub cls_def: ClassDefinition<'a>,
```

Comment:
Just a small nit: can we maybe call this `definition` instead?

Reply:
Upon reflection, I don't like the whole `ClassDefinitionWithHash` - now that I've finally fit the carpet, :-) it isn't really used that much...